frontend/src/components/dashboard/campaigns/Campaigns.tsx

```diff
@@ -63,18 +66,45 @@ const Campaigns = () => {
     setLoading(false)
   }
 
+  // Returns true if A < B, else false
+  function compareSemver(A: string, B: string) {
+    if (A == null) {
```

Comment:
```suggestion
    if (A === null) {
```

Reply:
I did `==` instead of `===` here to account for the case where the user wasn't found in the `user_features` table yet - hence A would be `undefined`. Is there a better way to handle that case?

Reply:
```suggestion
    if (!A) {
```

Maybe check for a [falsy](https://developer.mozilla.org/en-US/docs/Glossary/Falsy) value?

Reply:
Done!